node.go

```diff
@@ -382,3 +387,17 @@ func (n *ProtoNode) Tree(p string, depth int) []string {
 	}
 	return out
 }
+
+func ProtoNodeConverter(b blocks.Block, nd ipld.Node) (legacy.UniversalNode, error) {
+	pbNode, ok := nd.(dagpb.PBNode)
```

Comment:
is there a todo to add here to do codec stuff if it isn't dagpb, rather than failing?

Reply:
other codecs should "just work" based on the design of go-ipld-prime. Protobuf & Raw are special cases.